vmdb/app/models/mixins/password_mixin.rb

```diff
@@ -13,6 +18,9 @@ def encrypt_column(column)
       #   password_encrypted  : Get the password in cryptext
       #   password_encrypted= : Set the password in cryptext
 
+      encrypted_columns << column.to_s
+      encrypted_columns << "#{column}_encrypted"
```

Comment:
@gmcculloug in automate, do we do something similar to this?
(not trying to de-dup the code, just want it to behave similarly in both spots.)